src/backends/tensorrt/plan_backend.cc

```diff
@@ -172,11 +172,14 @@ PlanBackend::Context::Context(
   support_batching_ = (max_batch_size != NO_BATCHING);
 
   Status status = SupportsIntegratedZeroCopy(gpu_device_, &zero_copy_support_);
-  if ((!status.IsOk()) || (!zero_copy_support_)) {
-    LOG_INFO << "Zero copy optimization is disabled";
+  if (!status.IsOk()) {
+    LOG_ERROR << "Failed to check if zero copy is supported: "
+              << status.Message();
```

Comment:
Mention "Zero copy optimization is disabled" here as well, so user know this is not a fatal error.

Reply:
I prefer the failed to check. The wording itself makes it clear that it is just a check.

Reply:
Okay